addons/website_theme_install/models/theme_models.py

```diff
@@ -13,6 +14,8 @@ class ThemeView(models.Model):
     _description = 'Theme UI View'
 
     def compute_arch_fs(self):
+        if not self._context.get('install_filename'):
+            return False
```

Comment:
in wich case are we creating new template.theme without install context? seems louchhhhhhhhhhhhh

Reply:
oh right, this shouldn't be in this PR but maybe it should be considered as a fix. For my tests, I displayed the theme.ir.ui.menu records to translate them manually. The computed field crash with a KeyError when clicking on "create" button.

Reply:
The computed field ??

But it is a default value. 

So in my mind, only computed on create, and stored in db no?